internal/utils/handle_dependencies_test.go

```diff
@@ -0,0 +1,227 @@
+package utils
+
+import (
+	"fmt"
+	"regexp"
+	"runtime"
+	"testing"
+
+	"github.com/NethermindEth/1click/internal/pkg/commands"
+	"github.com/NethermindEth/1click/test"
+)
+
+func TestGetScriptPath(t *testing.T) {
+	inputs := []struct {
+		dependency string
+		isErr      bool
+	}{
+		{
+			"",
+			false,
+		},
+		{
+			"docker",
+			false,
+		},
+		{
+			"wR0n9",
+			false,
+		},
+	}
+
+	for _, input := range inputs {
+		descr := fmt.Sprintf("getScriptPath(%s)", input.dependency)
+
+		path, _, err := getScriptPath(input.dependency)
+		if input.isErr && err == nil {
+			t.Errorf("%s expected to fail.", descr)
+		} else if !input.isErr {
+			if err != nil {
+				t.Errorf("%s failed: %v", descr, err)
+			} else {
+				//TODO: improve results tests
+				match, err := regexp.MatchString(`.*`+input.dependency+`/.*/`+`\.sh`, path)
```

Comment:
Just curious, why ``.*`+input.dependency+`/.*/`+`\.sh`` instead of ``.*`+input.dependency+`/.*/\.sh``

Reply:
Is good to add a comment here describing what the regex does or what is its objective